### PR TITLE
finished the html-css project

### PR DESCRIPTION
**Pull request by Marbreeze**
Created everything like in instructions. It took so long because could't figure out the design, I have used webkit extension media query, flex-box mainly for CSS. I would appreciate any feedback!

### CODE REVIEW

**Review by Marbreeze — COMMENTED** [review]
fixed styling bug on iphone

**Review by ayunas — CHANGES_REQUESTED** [review]
Hi @Marbreeze .  I appreciate the work you put into the project.  I'm not able to approve the request because there are merge conflicts.  Could you fix the merge conflicts and @ayunas  me for review again once those are ironed out?  

Thanks!

**Comment by Marbreeze** [comment]
Git is saying there's no conflict with the base branch, also I pulled the master into my feature branch, I did not get any conflicts. Can you pin-point where exactly you see the conflict?

File: gallery.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+h1 {
+  font-family: "Montserrat", sans-serif;
+  color: orange;
+  font-size: 3em;
+  font-weight: 400;
+}
+
+@media screen and (min-width: 200px) {
```
Comment (ayunas) [inline gallery.css:72]: so you're doing mobile first design here?  and what device is less than 200px?  That is a very small screen size nowadays!

Comment (ayunas) [reply gallery.css:72]: I would recommend you do max-width because you're designing with the desktop in mind.

**Review by ayunas — APPROVED** [review]
Nice Effort Marina.  Code is looking fine overall, and appreciate the hard work you put into it.  

Regarding your design, the element sizes are too large to view on the screen.  The images and text should be scaled to nicely fit the viewport.  I do like the color choices with your web page though.

I would also leave some padding for the nav at the top.  The words are hitting the top of the page.

If possible also, I would try to fit the entire page in the viewport, so you don't have to scroll.  Your page isn't long enough that it needs a vertical scroll nor a horizontal scroll.

Also, you should usually avoid horizontal scrolling.  That leads to a bad user experience.

**Pull request merged**